cmd/logger/logger.go

```diff
@@ -155,6 +176,14 @@ func getTrace(traceLevel int) []string {
 			// Form and append a line of stack trace into a
 			// collection, 'trace', to build full stack trace
 			trace = append(trace, fmt.Sprintf("%v:%v:%v()", file, lineNumber, funcName))
+
+			// Ignore trace logs beyond the following conditions
+			if funcName == "http.HandlerFunc.ServeHTTP" ||
```

Comment:
Should we capture these constants perhaps in a string and then range through and match?

Reply:
done